script.js

```diff
@@ -1,4 +1,326 @@
+        100
+      ).toFixed(2);
+      myOutputValue =
+        myOutputValue +
+        ` It is a draw! Try again!<br><br>Your score: ${myScore} <br><br> Computer's score: ${computerScore} <br><br>Number of draws: ${numberOfDraws} <br><br> Your winning percentage: ${userWinningPercentage}% <br><br> Computer's winning percentage: ${computerWinningPercentage}%`;
+      return myOutputValue;
+    } else {
+      return `You can only input: <br> "scissors" <br> "paper" <br> "stone" <br> "reversed scissors" <br> "reversed paper" or <br> "reversed stone". <br> You have input something else, please try again!`;
+    }
+  } else if (currentGameMode == `korean`) {
+    var randomComputerMove = computerHand();
+
+    // Rules for first round:
+    if (
+      input == "scissors" &&
+      randomComputerMove == "paper" &&
+      myScore == 0 &&
+      computerScore == 0
+    ) {
+      myScore = myScore + 1;
+      computerScore = computerScore + 0;
+      return `${userName}, you threw ${input} and computer threw ${randomComputerMove}. You're the attacker. Go again!<br><br>Your score: ${myScore}`;
+    }
+
+    if (
+      input == "scissors" &&
+      randomComputerMove == "stone" &&
+      myScore == 0 &&
+      computerScore == 0
+    ) {
+      myScore = myScore + 0;
+      computerScore = computerScore + 1;
+      return `${userName}, you threw ${input} and computer threw ${randomComputerMove}. The computer is the attacker. Go again!<br><br>Your score: ${myScore}`;
+    }
+
+    if (
+      input == "paper" &&
+      randomComputerMove == "stone" &&
+      myScore == 0 &&
+      computerScore == 0
+    ) {
+      myScore = myScore + 1;
+      computerScore = computerScore + 0;
+      return `${userName}, you threw ${input} and computer threw ${randomComputerMove}. You're the attacker. Go again!<br><br>Your score: ${myScore}`;
+    }
+
+    if (
+      input == "paper" &&
+      randomComputerMove == "scissors" &&
+      myScore == 0 &&
+      computerScore == 0
+    ) {
+      myScore = myScore + 0;
+      computerScore = computerScore + 1;
+      return `${userName}, you threw ${input} and computer threw ${randomComputerMove}. The computer is the attacker. Go again!<br><br>Your score: ${myScore}`;
+    }
+
+    if (
+      input == "stone" &&
+      randomComputerMove == "scissors" &&
+      myScore == 0 &&
+      computerScore == 0
+    ) {
+      myScore = myScore + 1;
+      computerScore = computerScore + 0;
+      return `${userName}, you threw ${input} and computer threw ${randomComputerMove}. You're the attacker. Go again!<br><br>Your score: ${myScore}`;
+    }
+
+    if (
+      input == "stone" &&
+      randomComputerMove == "paper" &&
+      myScore == 0 &&
+      computerScore == 0
+    ) {
+      myScore = myScore + 0;
+      computerScore = computerScore + 1;
+      return `${userName}, you threw ${input} and computer threw ${randomComputerMove}. The computer is the attacker. Go again!<br><br>Your score: ${myScore}`;
+    }
```

Comment:
too many repeated code, if there are repeated code , it generally good to use function to extract it and reuse the function instead of typing the same logic.  if you make it into a function , the object "stone" ,"paper" and "scissors" can be change using the function parameter.

example 
var add = function(a,b){
    return a + b;
};
var result = add(" stone "," paper "); //expect to see "stone paper"
var result2 = add(" paper " ,"scissors" ); // expect to see " paper scissors"